_specification-v1/sass.md

```diff
@@ -96,12 +96,13 @@ As Sass has limited encapsulation, any Sass (e.g. mixin, function, variable) tha
 
 ## Sass Includes
 
-If a component contains SCSS files other than the main file listed in `bower.json`:
+If a component contains SCSS files other than the root `main.scss` file:
 
 - Files for import **must** be prefixed with an underscore, to [indicate privacy](#private-sass).
 - All import statements **must** be imported before any other Sass.
-- All import statements **should** be in the components’s main file.
+- All import statements **should** be in the components’s `main.scss` file.
 - Sass variables, mixins, and functions **should** be in their own files.
+- `_index.scss` **must** exist and only import the `main.scss` file.
```

Comment:
We should not include this, so that we can investigate sass modules.
We could introduce sass modules via the _index.scss file and still have the primary mixin feom the main.scs file
```suggestion
- `_index.scss` **must** exist and only import the `main.scss` file.
```